src/twisted/trial/_dist/test/test_workertrial.py

```diff
@@ -102,13 +102,16 @@ def test_forwardCommand(self):
         client = FakeAMP()
         clientTransport = StringTransport()
         client.makeConnection(clientTransport)
-        client.callRemote(workercommands.Run, testCase="doesntexist")
+        client.callRemote(workercommands.Run, testCase=b"doesntexist")
         self.readStream = clientTransport.io
         self.readStream.seek(0, 0)
         main(self.fdopen)
         self.assertIn(
             "No module named 'doesntexist'", self.writeStream.getvalue())
 
+    if _PY3:
+        test_forwardCommand.skip = "Does not work on Python 3 (https://tm.tl/8944)"
```

Comment:
This test doesn't seem too critical.  Omitting this test, but bringing more of the trial code into Python 3 allows more coverage.  Since trial is exercised by running all of the Twisted tests, that is a good indication that it is OK.